scripts/rollup/results.json

```diff
@@ -1,232 +1,346 @@
 {
   "bundleSizes": {
     "react.development.js (UMD_DEV)": {
```

Comment:
Let's just make the whole thing an array and add `filename` as one of the fields?

Reply:
Yep 👍